CONTRIBUTING.md

```diff
@@ -1,60 +1,144 @@
+- Make your changes
+- Build the documentation website to check your work (only possible if you are making changes locally)
+- Submit a [pull request](https://github.com/opensearch-project/documentation-website/pulls) (PR)
+- Maintainers review and merge your PR
 
-Before submitting, make sure to:
+### Before you start
 
-* Work against the latest source on the main branch.
-* Check existing open and recently merged pull requests to ensure that someone else hasn't addressed the problem already.
-* If the change requires significant work, open an issue where we can first discuss your request.
+Before contributing content, make sure to read the following resources:
+- [README](README.md)
+- [OpenSearch Project Style Guidelines](STYLE_GUIDE.md)
+- [API Style Guide](API_STYLE_GUIDE.md)
+- [Formatting Guide](FORMATTING_GUIDE.md) 
 
-To create a pull request:
+NOTE: Please make sure that any documentation you submit is your work or work that you have the rights to submit. We respect the intellectual property rights of others, and as part of contributing, we'll ask you to sign your contribution with a [Developer Certificate of Origin (DCO)](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#developer-certificate-of-origin) that states you have the rights to submit this work and you understand we'll use your contribution. 
 
-1. Fork the repository.
-2. Modify the source. Make sure to focus on the specific change you are contributing. For example, if you also reformat the code, it will be hard for us to focus on your change.
-3. Test the code. Make sure that any local tests pass.
-4. Commit to your fork using clear commit messages.
-5. Create the [pull request](https://github.com/opensearch-project/documentation-website/pulls) and save it, making sure to answer the default questions in the pull request template.
+### Making minor changes
 
-If you need additional support, GitHub provides documentation on [forking a repository](https://help.github.com/articles/fork-a-repo/) and [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
+If you want to make minor changes to an existing file, you can use this approach:
 
+1. [Fork this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
 
-## Finding contributions to work on
+1. In your fork on GitHub, navigate to the file that you want to change.
 
-If you’d like to contribute but don't know where to start, try browsing existing issues. Our projects use custom GitHub issue labels for status, version, type of request, etc., but we recommend looking at any issues labeled “good first issue” first. 
+1. In the upper-right corner, select the dropdown with the pencil icon and select **Edit in place**. Edit the file.
```

Comment:
Why do they have to use the "Edit in place" option? They can also just select the pencil icon, which defaults to editing through GitHub automatically?